tests/test_users_list.py

```diff
@@ -133,3 +116,27 @@ def test_list_unitusers_with_super_admin_correct_unit(client):
     unit_users = [x["Username"] for x in returned_users]
     for x in unit_row.users:
         assert x.username in unit_users
+
+
+def test_list_all_users_with_super_admin(client):
+    """Super admins need to specify a unit."""
```

Comment:
shouldn't this be something like

```suggestion
    """Super admins don't need to specify a unit."""
```
or
```suggestion
    """Super admins can list all user when no unit specified."""
```